openlibrary/core/imports.py

```diff
@@ -138,6 +138,22 @@ def mark_created(self, ol_key):
     def mark_modified(self, ol_key):
         self.set_status(status='modified', ol_key=ol_key)
 
+    @classmethod
+    def delete_items(
+        cls, ia_ids: list[str], sentinel_id: int = 321, _test: bool = False
+    ):
+        oldb = db.get_db()
+        data = {
+            'batch_id': sentinel_id,
+            'ia_ids': ia_ids,
+        }
+        return oldb.delete(
+            'import_item',
+            where=('batch_id > $batch_id AND ia_id IN $ia_ids'),
+            vars=data,
+            _test=_test,
+        )
+
```

Comment:
The sentinel ID may no longer be needed -- we can either restrict the ia_id deletion to a specific batch (based on the source_record) or we can just delete across the table.

Instead of sentinel, provide an optional batch_id (or default to all)